spec/features/questions/index_question_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+feature 'User see list of questions', '
+  I want to be able see questions
+' do
+
+  given(:user) { create(:user) }
+  given(:question) { create(:question, user: user) }
```

Comment:
"List" must contain at least 2 elements. Otherwise we cannot check that it behaves like a list.